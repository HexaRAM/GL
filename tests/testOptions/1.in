./analyseur tests/testOptions/1.lt -e