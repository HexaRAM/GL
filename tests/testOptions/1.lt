var x;
var y;
const a = 3;
const b = 12;
const c = 2+4*5+c, d=3, e=5;
ecrire a;
ecrire b/a;


