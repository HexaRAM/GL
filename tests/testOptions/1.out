# Programme lanc√© en mode EXECUTION (-e)
